client/app/components/about-modal/about-modal.controller.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+  'use strict';
+    
+  angular.module('app.components')
+    .directive('aboutModal', function() {
+      return {
+        templateUrl: "app/components/about-modal/about-modal.html",
+        controller: ModalCtrl,
```

Comment:
gonna be a stickler and request we call this `ModalController` (eslint is gonna bark in the near future anyways)